File: 2239/find-closest-number-to-zero.py
# 找到「最接近0」的數
class Solution:
    def findClosestNumber(self, nums: List[int]) -> int:
        ans = nums[0]  # 先隨便挑個數，當答案
        for num in nums:
            if abs(ans) > abs(num) or (abs(ans) == abs(num) and num > ans):
                ans = num
        return ans

File: 2239/find-closest-number-to-zero.cpp
class Solution {
public:
    int findClosestNumber(vector<int>& nums) {
        int ans = nums[0];
        for(int i=1; i<nums.size(); i++) {
            if(abs(nums[i])<abs(ans) || (abs(nums[i])==abs(ans)&&nums[i]>ans)) {
                ans = nums[i];
            }
        }
        return ans;
    }
};
